Error: Cannot find module 'http'

~https://github-code-review-notifier-1ev76qgm5.now.sh/_logs~

@styfle Sorry I thought the logs was public.😅

For the `gateway.js`:
I got `Cannot find module 'http'` error when I was using `@now/node`. Then I found an issue says that `dynamic require` might fail in some cases. So I made the `gateway.js` to test if that it was that caused the error.

Did you try paths other than `/`? I deployed again just now and other paths are still down :(

For example, [`/github`](https://github-code-review-notifier-6euhim87h.now.sh/github) throws 500:

<details><summary>Logs of that request</summary>

```
03/05 10:50 AM (7m)
(output/src/routes/github.js)
{ Error: Cannot find module 'http'
    at s (/var/task/user/src/routes/github.js:5283:262)
    at /var/task/user/src/routes/github.js:5283:419
    at Object.1.http (/var/task/user/src/routes/github.js:5283:601)
    at s (/var/task/user/src/routes/github.js:5283:368)
    at /var/task/user/src/routes/github.js:5283:419
    at Object.3../_read (/var/task/user/src/routes/github.js:5283:4286)
    at s (/var/task/user/src/routes/github.js:5283:368)
    at e (/var/task/user/src/routes/github.js:5283:534)
    at /var/task/user/src/routes/github.js:5283:554
    at module.exports.240.i (/var/task/user/src/routes/github.js:5283:61) code: 'MODULE_NOT_FOUND' }
```

</details>

PS: It does surprise me that you could find the source code repo and the right branch.🤣

@EnixCoda I am able to reproduce this bug.

1. Clone [this branch](https://github.com/EnixCoda/github-code-review-notifier/tree/feature/v2)
2. Change `now.json` env to the following:
```json
  {
    "FIREBASE": "eyJoZWxsbyI6ICJ3b3JsZCJ9",
    "VERIFICATION_TOKEN": "token",
    "CLIENT_SECRET": "client",
    "CLIENT_ID": "id",
    "SIGNING_SECRET": "secret",
    "NODE_ENV": "production"
  },
```
3. Run `ncc build src/github.js`
4. Run `node dist/index.js`

<details>

```
Error: Cannot find module 'http'
    at s (/Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:262)
    at /Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:419
    at Object.1.http (/Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:601)
    at s (/Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:368)
    at /Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:419
    at Object.3../_read (/Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:4286)
    at s (/Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:368)
    at e (/Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:534)
    at /Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:554
    at module.exports.364 (/Users/styfle/Desktop/github-code-review-notifier/dist/index.js:31777:61)
```

</details>

@guybedford Can you look into this bug?

--- a/package.json
+++ b/package.json
@@ -84,6 +84,7 @@
     "stripe": "^6.15.0",
     "terser": "^3.11.0",
     "the-answer": "^1.0.0",
+    "tiny-json-http": "^7.0.2",
     "ts-loader": "^5.3.1",
     "tsconfig-paths": "^3.7.0",
     "tsconfig-paths-webpack-plugin": "^3.2.0",

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "@google-cloud/firestore": "^0.19.0",
     "@sentry/node": "^4.3.0",
     "@tensorflow/tfjs-node": "^0.3.0",
-    "@zeit/webpack-asset-relocator-loader": "0.2.2",
+    "@zeit/webpack-asset-relocator-loader": "0.2.3",
     "analytics-node": "^3.3.0",
     "apollo-server-express": "^2.2.2",
     "arg": "^2.0.0",

--- a/test/integration/tiny-json-http.js
+++ b/test/integration/tiny-json-http.js
@@ -1,0 +1,1 @@
+const tiny = require('tiny-json-http');

